### DISCUSSION TITLE
Why we need two tokenizer in the train_dreambooth_lora_sdxl

### DISCUSSION THREAD

**Discussion by daiyixiang666** [huggingface/diffusers#7281] [Q&A]
I am a starter and I have a question that why we need two tokenizer to trian the lora in train_dreambooth_lora_sdxl, I am some of confused

**Comment by daiyixiang666** [comment]
We want to get the pooled embedding, I got this

**Comment by sayakpaul** [comment]
SDXL uses two different text encoders. That's why.
